### PR TITLE
Nov13

### PR DESCRIPTION
**Pull request by zeibura**
### What does this PR do?
A couple more minor changes to the Che4z installing page:
- link to devfile location reverted to code block showing the full path. A link implies that the user should download the file and this is not the case.
- link to Brightside documentation added, showing instructions for accessing the named PDF.

### What issues does this PR fix or reference?
15165

### CODE REVIEW

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 1]
```diff
@@ -25,9 +25,9 @@ This page describes how to install the Eclipse Che4z basic stack.
 
 . In a web browser, load `++https://++__<YOUR_CHE_HOST>__/f?url=__<DEV_FILE_LOCATION>__`.
 +
-* For the Eclipse Che4z basic stack, use the link:https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml[`mainframe-basic-stack.yaml`] devfile.
+* For the Eclipse Che4z basic stack, use the devfile `https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml`
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]: Reason for this? We try to avoid naked URLs -- why is this needed? Also, using the `link:` notation is good practice for live links. As for word order, "use the foo devfile" is IMO better than "use the devfile foo".

Comment (zeibura) [reply src/main/pages/che-7/extensions/che4z-installing.adoc]: We provide the URL so that the user can copy and paste it, not so the user can click the link and download the file. Surely a codeblock is preferable to a link for this purpose?

Comment (zimlu02) [reply src/main/pages/che-7/extensions/che4z-installing.adoc]: Zeb is right - we don't want users to click on the link, we want them only to copy the string with URL.

Comment (rkratky) [reply src/main/pages/che-7/extensions/che4z-installing.adoc]: Ok, in that case, the link should not be live.
```suggestion
* For the Eclipse Che4z basic stack, use the `++https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml++` devfile.
```

File: src/main/pages/che-7/extensions/che4z-installing.adoc [hunk 2]
```diff
@@ -17,17 +17,17 @@ This page describes how to install the Eclipse Che4z basic stack.
...
 
 . Log in to Eclipse Che.
 
 . In a web browser, load `++https://++__<YOUR_CHE_HOST>__/f?url=__<DEV_FILE_LOCATION>__`.
 +
-* For the Eclipse Che4z basic stack, use the link:https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml[`mainframe-basic-stack.yaml`] devfile.
+* For the Eclipse Che4z basic stack, use the devfile `https://github.com/eclipse/che-che4z/raw/1.0.0/mainframe-basic-stack.yaml`
 
-* For a purchased premium stack, use the devfile location specified in the PDF installation guide available from link:https://casupport.broadcom.com/download-center/download-center.html[Broadcom Support].
+* For a purchased premium stack, use the devfile location specified in the PDF installation guide available from Broadcom Support. Refer to the link:http://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside-enterprise/2-0/release-notes.html[Brightside documentation] ("Getting Started" section) for instructions on accessing the PDF.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-installing.adoc]:
```suggestion
* For a purchased premium stack, use the devfile location specified in the PDF installation guide available from Broadcom Support. See the link:http://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-mainframe-software/devops/ca-brightside-enterprise/2-0/release-notes.html[Brightside documentation] (the "Getting Started" section) for instructions on accessing the PDF.
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by zeibura** [comment]
That's fine by us.

**Review by rkratky — APPROVED** [review]
